packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageList.tests.js

```diff
@@ -125,6 +135,153 @@ QUnit.module('MessageList', moduleConfig, () => {
 
             assert.strictEqual($messageGroups.length, 26);
         });
+
+        [
+            {
+                date: new Date(),
+                scenario: 'today\'s date',
+                text: 'Today ',
+            },
+            {
+                date: new Date(Date.now() - MS_IN_DAY),
+                scenario: 'yesterday\'s date',
+                text: 'Yesterday ',
+            },
+            {
+                date: new Date(Date.now() - 2 * MS_IN_DAY),
+                scenario: 'some older date',
+                text: '',
+            },
+        ].forEach(({ date, scenario, text }) => {
+            QUnit.test(`Day header should be rendered (message date is ${scenario}, on init)`, function(assert) {
+                const expectedText = `${text}${getStringDate(date)}`;
+
+                this.reinit({ items: [{ timestamp: date, text: 'ABC' }] });
+
+                const scrollableContent = this.getScrollable().content();
+                const $dayHeader = $(scrollableContent).find(`.${CHAT_MESSAGELIST_DAY_HEADER_CLASS}`);
```

Comment:
Can we extract this into helper functions to module level?

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925